packages/blocks/image/src/Image.tsx

```diff
@@ -198,13 +199,16 @@ export const Image: BlockComponent<AppProps> = (props) => {
       file?: FileType;
     }) => {
       if (src?.trim()) {
-        if (updateEntities && entityId) {
+        if (updateEntities && accountId && entityId) {
```

Comment:
`accountId` is optional in blocks - it should not be required to call an update. This applies in a few other places.

It is required in the `hash` app - so we should handle its absence in the hooks (as you do). But the blocks should work elsewhere, and other apps might not have `accountId`.

Reply:
Fixed here: https://github.com/hashintel/hash/pull/291/commits/f2fbcc726dd069b54a93b17877bcc14fe47ef52f